module/zfs/zil.c

```diff
@@ -92,6 +92,11 @@
  */
 static uint_t zfs_commit_timeout_pct = 5;
 
+/*
+ * Minimal time we care to delay commit waiting for more ZIL records.
+ */
+static uint64_t zil_min_commit_timeout = 5000;
```

Comment:
could you add some comments about how/why we chose this default value? I think some of that might be in the PR description, but it'd be good to put that in a code comment here, or maybe the man page that describes the tunable?

Reply:
perfect, thanks.